src/test/java/org/folio/rest/impl/OrdersResourceImplTest.java

```diff
@@ -125,23 +126,35 @@ public void testPoCreationFailure(TestContext ctx) throws Exception {
 
     String body = getMockData(poCreationFailurePath);
 
-    final Response resp = RestAssured
-      .with()
-      .header(X_OKAPI_URL)
-      .header(X_OKAPI_TENANT)
-      .contentType(APPLICATION_JSON)
-      .body(body)
-      .post(rootPath)
-      .then()
-      .contentType(TEXT_PLAIN)
-      .statusCode(400)
-      .extract()
-      .response();
+    final Async asyncLocal = ctx.async();
 
-    String respBody = resp.getBody().asString();
-    logger.info(respBody);
-
-    assertEquals("Invalid po_number", respBody);
+    final Errors errors = RestAssured
+      .given()
+        .header(X_OKAPI_URL)
+        .header(X_OKAPI_TENANT)
+        .contentType(APPLICATION_JSON)
+        .body(body)
+      .when()
+        .post(rootPath)
+      .then()
+        .log()
+          .all()
+        .contentType(APPLICATION_JSON)
+        .statusCode(422)
```

Comment:
It's pretty cool that you get the 422/Unprocessible Entity for free... I was unaware of that and was confused when I saw this until I looked at the RMB code.

Reply:
Yeah, pretty much anything that causes schema validation errors is reported as a 422 by Okapi. Sorry for the confusion.